### DISCUSSION TITLE
Translation Issue: tips when searching code in the repo

### DISCUSSION THREAD

**Discussion by Ivlyth** [community/community#84178] [Mobile]
### Select Topic Area

Product Feedback

### Body

when searching code in the repo, it will show up a tips like the screenshot below. 

![image](https://github.com/community/community/assets/5511286/b73d6b65-c54e-4e18-9281-ffdf53837944)

instead of &nbsp;&nbsp;&nbsp;`使用功能强大的 apache/dubbo-go 在 代码搜索语法 中搜索代码`, 
it should be `使用功能强大的 代码搜索语法 在 apache/dubbo-go 中搜索代码`

(swap the function name and repo name)

**Comment by esahmed21** [comment]
Hi @Ivlyth 👋 

Thank you for bringing this bug to our attention! We'll open an internal ticket to address it soon. Feel free to contact us for any other issues.

Thanks again for your assistance
